mscore/editstaff.ui

```diff
@@ -422,16 +509,56 @@
           </property>
          </widget>
         </item>
-        <item row="1" column="1">
-         <widget class="QTextEdit" name="shortName">
-          <property name="minimumSize">
-           <size>
-            <width>0</width>
-            <height>40</height>
-           </size>
+        <item row="3" column="0">
+         <widget class="QLabel" name="labelOffsetX">
+          <property name="text">
+           <string>Name Offset X:</string>
           </property>
-          <property name="tabChangesFocus">
-           <bool>true</bool>
+         </widget>
+        </item>
+        <item row="2" column="0">
+         <widget class="QLabel" name="rotationLabel">
+          <property name="text">
+           <string>Name rotation:</string>
+          </property>
+         </widget>
+        </item>
+        <item row="2" column="1">
+         <widget class="QSpinBox" name="rotationInput">
+          <property name="minimum">
+           <number>-90</number>
+          </property>
+          <property name="maximum">
+           <number>90</number>
```

Comment:
wouldn't +/-180 make sense too?

Reply:
and a tooltip about what turns clockwise and what counter clock wise

Reply:
@Jojo-Schmitz 

> wouldn't +/-180 make sense too?

The letters would be upside down, wouldn't they?

> and a tooltip about what turns clockwise and what counterclockwise

Sounds good. I should probably add tooltips for the offsets too.

Reply:
yes, 180° is upside down